lib/licensed/source/bundler.rb

```diff
@@ -57,9 +60,8 @@ def lockfile_path
       def with_local_configuration
         # with a clean, original environment
         ::Bundler.with_original_env do
-          # bundler restores all ENV at the end of the `with_original_env`
-          # block.  we shouldn't need to restore these values manually
-          BUNDLER_ENV_KEYS.each { |k| ENV.delete(k) }
+          # force bundler to use the local gem file
+          ENV["BUNDLE_GEMFILE"] = gemfile_path.to_s
```

Comment:
@jonabc Should this `ENV` var be unset in the `ensure` block below? Otherwise this method that takes a block leaks that environment variable?

Reply:
@dbussink It's not necessary and won't leak because `::Bundler.with_original_env` captures and restores the full `ENV` object itself.  👍 to restoring the `ENV` var in the `ensure` block to be more explicit and avoid confusion

Reply:
Ah, thanks! Yeah, explicitly doing it is nice I think.